ngrinder-controller/src/main/java/org/ngrinder/infra/config/Config.java

```diff
@@ -791,4 +792,12 @@ public String getCsvSeparator() {
 		return csvSeparator;
 	}
 
+	public boolean ignoreStop(StopReason stopReason) {

```

Comment:
I think this is not appropriate locate this in this Config.
What bout moving this to PerfTestRunnable?
Stop reason determination role should be PerfTestRunnable.

Reply:
thanks for the advice!